src/payment/v1/payment-mappers/payment-mapper.js

```diff
@@ -45,7 +45,7 @@ export default class PaymentMapper {
             notify_url: order.callbackUrl,
             return_url: paymentMethod.returnUrl || (order.payment ? order.payment.returnUrl : null),
             vault_payment_instrument: !payment.instrumentId ? payment.shouldSaveInstrument : null,
-            set_as_default_stored_instrument: (payment.instrumentId || payment.shouldSaveInstrument) ? payment.setAsDefaultInstrument : null,
+            set_as_default_stored_instrument: (payment.instrumentId || payment.shouldSaveInstrument) ? payment.shouldDefaultInstrument : null,
```

Comment:
`shouldSetDefaultInstrument` feels a little more descriptive

Reply:
changed to `shouldSetAsDefaultInstrument `